FamilySearchIosSampleApp/AncestorDetails.swift

```diff
@@ -36,68 +36,71 @@ class AncestorDetails : UIViewController
+                                            self?.ancestorImageIndicator.hidesWhenStopped = true
+                                            self?.ancestorDataIndicator.stopAnimating()
+                                            self?.ancestorDataIndicator.hidden = true
                                             
                                             // birth data
-                                            self.ancestorBirthLabelTitle.text = NSLocalizedString("ancestorDetailsBirth", comment: "Birth:")
-                                            self.ancestorBirthLabelTitle.hidden = false
-                                            self.ancestorBirthLabelValue.text = personDetails?.personBirthDate
-                                            self.ancestorBirthLabelValue.hidden = false
+                                            self?.ancestorBirthLabelTitle.text = NSLocalizedString("ancestorDetailsBirth", comment: "Birth:")
+                                            self?.ancestorBirthLabelTitle.hidden = false
+                                            self?.ancestorBirthLabelValue.text = personDetails?.personBirthDate
+                                            self?.ancestorBirthLabelValue.hidden = false
                                             
                                             // death data
                                             if (personDetails?.personDeathDate != nil)
                                             {
-                                                self.ancestorDeathLabelTitle.text = NSLocalizedString("ancestorDetailsDeath", comment: "Death:")
-                                                self.ancestorDeathLabelTitle.hidden = false
-                                                self.ancestorDeathLabelValue.text = personDetails?.personDeathDate
-                                                self.ancestorDeathLabelValue.hidden = false
+                                                self?.ancestorDeathLabelTitle.text = NSLocalizedString("ancestorDetailsDeath", comment: "Death:")
+                                                self?.ancestorDeathLabelTitle.hidden = false
+                                                self?.ancestorDeathLabelValue.text = personDetails?.personDeathDate
+                                                self?.ancestorDeathLabelValue.hidden = false
                                             }
                                         })
                                     }
                                 })
     }
     
-    func getAncestorDetailsData(personUrlString:String,
+    func getAncestorDetailsData(personUrlString:String?,
                                 accessToken:String,
                                 completionAncestorDetails:(responseDetails: PersonDetails?, reponseError:NSError?) -> ())
     {
+        let ancestorDetailsUrl = NSURL(string: personUrlString!);
```

Comment:
We could get rid of our force unwrapped optional here by doing:
```
guard let personUrlString = personUrlString, ancestorDetailsUrl = NSURL(string: personUrlString) else {
    //Explicitly handle bad url creation here.
}
```
That would also get rid of the force unwrapping on line 89.